wasm.md

```diff
@@ -15,10 +15,10 @@ Configuration
 
 ```k
     configuration
-      <wasm>
+      <wasm unused="">
         <instrs> .K </instrs>
         <valstack> .ValStack </valstack>
-        <curFrame>
+        <curFrame unused="">
           <locals>    .Map </locals>
           <curModIdx> .Int </curModIdx>
```

Comment:
Why do we need `unused` here? There are definitely rules which use the `<locals>` cell under the `<curFrame>` cell. I don't think `<curFrame>` itself is ever used directly in a rule, but we should definitely not count it as "unused" because its child cells _are_ used.

@dwightguth thoughts?

Reply:
It is in fact an unused symbol; you declare it in the configuration but never reference it anywhere in any of your rules. You could remove this cell from the configuration and place its children under their parent without any change in the semantics. Thus the warning.

Reply:
I'm wondering if it makes sense to exclude the top-level cell from this warning, or at least any cell which is used in a composition. For example, the <wasm> cell is used for configuration composition inside the <wasm-text> cell, so that should probably not be seen as "unused". In general, one may want a top-level cell in one's semantics, rather than the <generatedTop> one.

Reply:
I mean, it's used in the sense that the compiled definition makes use of it, but not used in the sense that it can be removed without changing the behavior of the semantics in any way. Personally I think the latter meaning is what we usually think of when we think of unused code.

Reply:
I agree when it comes to the <curFrame> cell, but what about the <wasm> cell? It is at the very least used for composition, and as such could not be removed without breaking the definition.

Reply:
I think you are right that we should in some sense view the top cell of an individual configuration declaration as "externally visible" and thus exclude it from this warning, because it might be unused in one semantics with the intent that it is used in another semantics it is composed from. If you create an issue for this and get it assigned to me, I will fix it tomorrow. It should be a very straightforward fix.

Reply:
Sounds like a good compromise. There is definitely some design space here, but in lieu of mechanisms in K for explicitly exporting/hiding some parts of a module this is the right thing. I mean, why would you even add a named top-level cell if not to compose/avoid the automatic `<generatedTop>` cell?

One other thing that doesn't sit right is the choice of word. `unsued` seems to miss the mark a bit. Like @ehildenb said, it's not really "unused", rather, it is just notational. It's contents gets modified. At least conceptually, the configuration is just a way to structure data, and as such the grouping and naming is expected to be documentary. Maybe a keyword like `structural` would make more sense?